source/postprocess/particles.cc

```diff
@@ -705,6 +705,9 @@ namespace aspect
             }
 
           exclude_output_properties = Utilities::split_string_list(prm.get("Exclude output properties"));
+
+          // Never output the integrator properties that are for internal use only
+          exclude_output_properties.push_back("integrator properties");
```

Comment:
This is where it would be useful to mark this field as "internal:" or some such.